reference/configuration/framework.rst

```diff
@@ -1506,11 +1506,24 @@ errors.
 cache_dir
 .........
 
+.. deprecated:: 7.1
```

Comment:
Usually, this section is placed at the end of the option description.

Reply:
As 7.0 was released recently, there's not so many examples in the current branch 😅 But corrected 👍